alacritty/src/input.rs

```diff
@@ -514,8 +513,8 @@ impl<'a, T: EventListener, A: ActionContext<T>> Processor<'a, T, A> {
             );
             self.ctx.mouse_mut().scroll_px += new_scroll_px * multiplier;
 
-            let cmd = if new_scroll_px > 0 { b'A' } else { b'B' };
-            let lines = (self.ctx.mouse().scroll_px / height).abs();
+            let cmd = if new_scroll_px > 0.0 { b'A' } else { b'B' };
```

Comment:
```suggestion
            let cmd = if new_scroll_px > 0. { b'A' } else { b'B' };
```

Reply:
thanks for the nightly information. its all properly fomratted now.